pkg/kyma/cmd/version.go

```diff
@@ -27,9 +27,14 @@ func NewVersionOptions(o *core.Options) *VersionOptions {
 func NewVersionCmd(o *VersionOptions) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "version",
-		Short: "Version of the kyma CLI and connected Kyma cluster",
-		Long: `Prints the version of kyma CLI itself and the version of the kyma cluster connected by current KUBECONFIG
-`,
+		Short: "Version of the Kyma CLI and connected Kyma cluster",
+		Long: `Prints the version of the Kyma CLI and the version of the Kyma cluster the current KUBECONFIG points to.
+
```

Comment:
Prerequisites:
- (Optional) Kubernetes cluster is available with your KUBECONFIG already pointing to it.